### PR TITLE
[W5.11][F14-A1]Ahmad Syafiq

### PR DESCRIPTION
**Pull request by ahmadsyafiqruhazat**
LO-2KLoC

- Find command now ignores case (not case-sensitive). "find john" returns "John"."john","JOHN".
- Updated documentation and tests.

### CODE REVIEW

File: src/seedu/addressbook/commands/FindCommand.java [hunk 1]
```diff
@@ -50,11 +51,22 @@ public CommandResult execute() {
...
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            if (!Collections.disjoint(toLowerCase(wordsInName), toLowerCase(keywords))) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
+    /**
+     * Changes words into lower case
```
Comment (takuyakanbr) [inline src/seedu/addressbook/commands/FindCommand.java:61]: Javadoc comments, including the tags, should end with a full-stop.

File: doc/UserGuide.md [hunk 2]
```diff
@@ -57,12 +57,12 @@ Format: `list`
 Finds persons whose names contain any of the given keywords.<br>
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
-> The search is case sensitive, the order of the keywords does not matter, only the name is searched, 
+> The search is not case sensitive, the order of the keywords does not matter, only the name is searched,
```
Comment (takuyakanbr) [inline doc/UserGuide.md:60]: You can consider using 'case insensitive' instead of 'not case sensitive'.

**Review by takuyakanbr — CHANGES_REQUESTED** [review]
Good work! Some minor style issues with the javadoc comment for the added method.

**Review by cjianhui — COMMENTED** [review]
@lithiumlkid Good job on updating the documentation and tests. Apart from minor issues as mentioned by Siyu, everything else looks fine. 

You may close the PR.

**Pull request closed without merging**